.ci/static-checks.sh

```diff
@@ -23,10 +23,23 @@ files_to_remove=()
 
 script_name=${0##*/}
 
+# Static check functions must follow the following naming conventions:
+#
+
+# All static check function names must match this pattern.
+typeset -r check_func_regex="^static_check_"
+
+# All architecture-specific static check functions must match this pattern.
+typeset -r arch_func_regex="_arch_specific$"
+
 repo=""
 specific_branch="false"
 force="false"
 branch=${branch:-master}
+disable_arch_specific="false"
+only_arch_specific="false"
```

Comment:
These two `arch_specific` vars sound like the inverse of each other - can they be melded into one and some of the tests that use them inverted to match?

Reply:
Fixed.